worlds/ladx/LADXR/.github/workflows/multiworld_build.yml

```diff
@@ -0,0 +1,36 @@
+name: Build multiworld client
```

Comment:
I dont think you need to include this build pipeline

Reply:
addressed